shimming-toolbox/shimmingtoolbox/cli/b0shim.py

```diff
@@ -1460,7 +1460,152 @@ def max_intensity(fname_input, fname_mask, fname_output, verbose):
     logger.info(f"Txt file is located here:\n{fname_output}")
 
 
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--anat', 'fname_anat', nargs=1, type=click.Path(exists=True), required=False,
+              help="Target image to shim. Supported formats: .nii, .nii.gz")
+@click.option('-i', '--input', 'fname_input', nargs=1, type=click.Path(exists=True), required=True,
+              help="Text file containing the shim coefficients. Supported formats: .txt")
+@click.option('-i2', '--input2', 'fname_input2', nargs=1, type=click.Path(exists=True), required=True,
+              help="Text file containing the shim coefficients. Supported formats: .txt")
+@click.option('--input-file-format', 'i_format',
+              type=click.Choice(['slicewise', 'chronological', 'custom-cl']), required=True,
+              help="Syntax used to describe the sequence of shim events for a coil or coil channel. "
+                   "Use 'slicewise' to output in row 1, 2, 3, etc. the shim coefficients for slice "
+                   "1, 2, 3, etc. Use 'chronological' to output in row 1, 2, 3, etc. the shim value "
+                   "for trigger 1, 2, 3, etc. The trigger is an event sent by the scanner and "
```

Comment:
Chronological is in the order that the slices are acquired on the scanner chronologically. This means that if there is one trigger per slice, but a slice order that is interleaved, it will not be the same as slice-wise but will have the same number of lines.

Custom-cl is the custom format used by our GE collaborator.

Reply:
I would add it in the description in the cli inputs